### DISCUSSION TITLE
Could I connect to DFP-34X-2C2 to SFP port of a unmanaged switch?

### DISCUSSION THREAD

**Discussion by mbuzzo1** [Anime4000/RTL960x#246] [Q&A]
I'm trying replace Media Converter with a unmanaged switch with 24 ports + SFP ports. Could I replace the Media Converter with that?

My question is related with if is possible that DFP-34X-2C2 works in a unmanaged switch and if it is possible, then, how I could connect to SFP device IP.

Also, if that is possible, I could manage the VLANs on the Router and then back with the VLAN tagged to the switch itself?

**Comment by Anime4000** [comment]
This seem bad idea, if you plan re-use that switch, it will expose VLAN from GPON to your network. Most GPON stick will automatically bridge VLAN what OLT give
